pkg/controller/logstorage/common/common.go

```diff
@@ -46,8 +45,8 @@ const (
 // are generated and stored in the user secret, a hashed version of the credentials is stored in the tigera-elasticsearch namespace for ES Gateway to retrieve and use to compare
 // the gateway credentials, and a secret containing real admin level credentials is created and stored in the tigera-elasticsearch namespace to be swapped in once
 // ES Gateway has confirmed that the gateway credentials match.
-func CreateKubeControllersSecrets(ctx context.Context, esAdminUserSecret *corev1.Secret, esAdminUserName string, cli client.Client) (*corev1.Secret, *corev1.Secret, *corev1.Secret, error) {
```

Comment:
I think we ultimately want to get rid of this, and create secrets via the ECK API instead of having kube-controllers do it. Then, we can get rid of es-kube-controllers access to Elasticsearch entirely (and also get rid of es-gateway).

Reply:
For now I have actually skipped this for multi-tenant, so I could theoretically revert the changes to this function and only call it for single-tenant clusters.